homeassistant/components/zha/number.py

```diff
@@ -970,6 +970,21 @@ class ThermostatLocalTempCalibration(ZHANumberConfigurationEntity):
     _attr_native_unit_of_measurement: str = UnitOfTemperature.CELSIUS
     _attr_icon: str = ICONS[0]
 
+    def __init__(
+        self,
+        unique_id: str,
+        zha_device: ZHADevice,
+        cluster_handlers: list[ClusterHandler],
+        **kwargs: Any,
+    ) -> None:
+        """Init this ZHA local temperature calibration entity."""
+        self._cluster_handler: ClusterHandler = cluster_handlers[0]
+        super().__init__(unique_id, zha_device, cluster_handlers, **kwargs)
+        if zha_device.model == "TRVZB":
```

Comment:
we should just add a custom ThermostatLocalTempCalibration implementation w/ these values. Extend that class and override the values. On the new class add the matching params for the model. Here is an example: https://github.com/home-assistant/core/blob/d004011d415707ce85cf657c024ff248d7bc2caf/homeassistant/components/zha/sensor.py#L406 you will want to add stop_on_match_group to both classes too. LMK if you have any issues.

Reply:
I was not sure how to name this class so it is now called `ThermostatLocalTempCalibrationSonoffTRVZB`. Is this correct? Because I can't have two classes called `ThermostatLocalTempCalibration`, correct?

Reply:
> I can't have two classes called ThermostatLocalTempCalibration, correct?

Correct. Your name is probably fine. Generally, we rather put the manufacturer and model in front though.
So, you could change it to something like `SonoffTrvThermostatLocalTempCalibration` perhaps?
But the TRV part can be omitted, since that's clear, so maybe use: `SonoffThermostatLocalTempCalibration` (?)